platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -163,11 +167,9 @@
     /**
      * Constructs the list of DDF system Settings (read-only and configurable
      * settings) to be pushed to registered ConfigurationWatchers.
-     * 
-     * @param services
-     *            the list of watchers of changes to the DDF System Settings
-     * @param configurationAdmin
-     *            the OSGi Configuration Admin service handle
+     *
```

Comment:
Comments don't format the same way for me in Eclipse and I'm using the standard format preference file that's on the DDF Developer's Guide. Any idea why your formatting is different?